docs/src/pages/guides/cli-options.md

```diff
@@ -13,7 +13,7 @@ monika -h
 
 ## Configuration
 
-Monika by default will look for the `monika.json` config file.
+Monika by default will look for the `monika.yaml` config file.
```

Comment:
this will need to change the getDefaultConfig() in index.ts file, to search for yaml file first then check the json config. Please change to search for this condition first :
: monikaDotYamlFile
    ? `./${monikaDotYamlFile}`

Reply:
good point. changed the order for config searched.